### PR TITLE
Update README with asdf installation method

### PR DESCRIPTION
**Pull request by THenry14**
Partially implements #111 

## Introduced changes

- adds information on asdf installation steps to readme

## Checklist

- [X] Linked relevant issue
- [X] Updated relevant documentation (README.md)
- [ ] Added relevant tests
- [ ] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: README.md
Comment (ksew1) [inline README.md:1]: Two things:
- Is asdf for the profiler already set up?  
- Perhaps we should say that we do not recommend script-based installation and instead recommend installation via asdf.

Comment (THenry14) [reply README.md:1]: ad 1. yup, it's already working as described in this PR. Still awaiting approval for shorthand plugin installation https://github.com/asdf-vm/asdf-plugins/pull/1074, but this is non blocking in any way

ad 2. do you have something specific in mind? right now readme says `You can install `cairo-profiler` using asdf (recommended) or the installation script.`, do you want it to be more promiment or something?

Comment (ksew1) [reply README.md:1]: Sorry, I didn't see the 'recommended.' Maybe we should include it in `### asdf` so it stands out more and is more noticeable.

**Review by ksew1 — APPROVED** [review]

File: README.md
Comment (THenry14) [reply README.md:1]: sure, it may be more visible there. changed :+1:

**Pull request merged**